workshops/views.py

```diff
@@ -917,6 +916,63 @@ def export(request, name):
 
 #------------------------------------------------------------
 
+@login_required
+def workshops_over_time(request):
+    '''Export CSV of count of workshops vs. time.'''
+
+    data = dict(Event.objects.past_events().\
+                    values_list('start').\
+                    annotate(Count('id')).\
+                    order_by('start'))
```

Comment:
Don't break lines like this. If long lines are wrapped in brackets then you don't need to use `\`. Additionally you should put `.method2()` aligned to `.method1()` in preceding line. So this snipped would look like this:

```python
    data = dict(Event.objects.past_events()
                     .values_list('start')
                     .annotate(Count('id'))
                     .order_by('start'))
```

Additionally, as I mentioned earlier, we don't need to `order_by` after all, because dict doesn't keep its data ordered.